ingestors/email_ingestor/email_ingestor.py

```diff
@@ -60,6 +65,11 @@ def _ingest_email_row(self, row: pd.Series) -> List[Document]:
             'date': row['date']
         }
 
+        # Replacing None values {None: ""}
+        for key in email_doc.metadata:
+            if email_doc.metadata[key] is None:
+                email_doc.metadata[key] = ""
+
```

Comment:
Issues ingesting to vectorDB where caused by metadata.key = None for some documents

I noticed it to be specifically caused when 'to' is blank and sender instead is sending privately to multiple recipients with 'bcc'